controllers/fenceagentsremediation_controller_test.go

```diff
@@ -648,6 +650,52 @@ var _ = Describe("FAR Controller", func() {
 				verifyEvent(corev1.EventTypeNormal, utils.EventReasonRemoveOutOfServiceTaint, utils.EventMessageRemoveOutOfServiceTaint)
 			})
 		})
+
+		When("A FAR remediation that was timed out by NHC is deleted", func() {
```

Comment:
Is this part of this PR?

Reply:
Good catch. Those test belong to another PR.